exercises/concept/lucas-numbers/lucas_numbers.rb

```diff
@@ -0,0 +1,14 @@
+class LucasNumbers
+  def initialize
+    raise NotImplementedError, "please implement the LucasNumbers #initialize method"
```

Comment:
```suggestion
    raise NotImplementedError, "please implement the LucasNumbers#initialize method"
```

Reply:
Question: does a student _have_ to define the `initialize` method? In theory, they could also define a method named `sequence` and it would still work? I wonder if we should somehow guide the student a bit. Also, I assume students know about `:attr_reader` at this point?